### ISSUE TITLE
Make documentation clear for beginners

### ISSUE THREAD

**Issue by mrranger** [popos123/Logitech-G29-Motherboard-Replacement#1]
It’s not at all clear what an Arduino or STM32 is needed, there’s a bit of a lack of photos, what boards are needed, it’s not clear in the documentation, I’d like to see photos of the wiring roughly like in the picture, then all the questions would disappear, well, a little tweaking of what to put where and what firmware .
![mpu-sch-500x288](https://github.com/popos123/Logitech-G29-Motherboard-Replacement/assets/13420378/f25fe5ea-fff2-4b0d-bdeb-c99bd62ff372)

In general, the project is excellent, it will help those whose main g29 board is broken

**Comment by popos123** [comment]
yeah i know, im preparing the cleaner, simpler version, sorry for that mess :(

**Comment by ranibrother** [comment]
Hi, I have some questions about the project:
1 - Would the exchange of the stm32f103 for the stm32f401 be the main board or the secondary one? Would the pinout be different?
2 - Are the encoder and virtual buttons on the electrical diagram optional?
3 - I will modify the motors and encoder to one with 600 pulses, should the code be different?
+ I have the original board, if you want I can send you the original firmware, I just don't know how to do that

**Comment by ranibrother** [comment]
> sim, eu sei, estou preparando uma versão mais limpa e simples, desculpe pela bagunça :(

Olá, tenho algumas dúvidas sobre o projeto:
1 - A troca do stm32f103 pelo stm32f401 seria a placa principal? O secundário segue o mesmo esquema quando usado 2 f103?
2 - O encoder virtual e os botões virtuais do diagrama elétrico são opcionais?
3 - Vou modificar os motores e o encoder para um com 600 pulsos, o código deve ser diferente?
4 - O arquivo g29emu.bin deve ser carregado na placa secundária?
5 - No esquema f401, os pinos 3 e 8 do shifter devem ser ligados em 3,3v, enquanto que no esquema f103, os 3,3v são ligados nos pinos 3 e 7. Qual é o correto?

Eu tenho a placa original, se quiser posso te enviar o firmware original, só não sei como fazer isso

**Comment by ranibrother** [comment]
> yeah i know, im preparing the cleaner, simpler version, sorry for that mess :(

Hello, I have some questions about the project:
1 - Would replacing the stm32f103 with the stm32f401 be the main board? Does the secondary board follow the same scheme when using 2 f103s?
2 - Are the virtual encoder and virtual buttons in the electrical diagram optional?
3 - I will modify the motors and the encoder to one with 600 pulses, should the code be different?
4 - Should the g29emu.bin file be loaded onto the secondary board?
5 - In the f401 scheme, pins 3 and 8 of the shifter should be connected to 3.3v, while in the f103 scheme, the 3.3v are connected to pins 3 and 7. Which one is correct?

I have the original board, if you want I can send you the original firmware, I just don't know how to do it

**Comment by kn0xx01** [comment]
yep.. I'm novice in these eletronics readings ... but a better guide on to achieve this with arduino would be much appreciated. 
my G29 board got broken.
